src/pages/folder/[folderId].tsx

```diff
@@ -0,0 +1,53 @@
+import { useRouter } from 'next/router';
+import { useEffect } from 'react';
+import useRequest from '@/hooks/useRequest';
+import { Folder as IFolder, Link } from '@/types/Folder.types';
+import Folder from '@/containers/Folder';
+
+function FolderPage() {
+  const router = useRouter();
+  const folderId = Array.isArray(router.query.folderId)
+    ? router.query.folderId[0]
+    : router.query.folderId;
+
+  const { data: folders, fetch: getFolders } = useRequest<{
+    data: { folder: IFolder[] };
+  }>({
+    skip: true,
+    options: {
+      url: `folders`,
+      method: 'get',
+    },
+  });
+  const { data: links, fetch: getLinks } = useRequest<{
+    data: { folder: Link[] };
+  }>({
+    skip: true,
+    options: {
+      url: `links`,
+      method: 'get',
+      params: { folderId: folderId ?? 1 },
+    },
+  });
+
+  useEffect(() => {
+    const accessToken = localStorage.getItem('accessToken');
+    if (!accessToken) {
+      router.push('/signin');
+    }
+    getFolders({ headers: { Authorization: `Bearer ${accessToken}` } });
+    getLinks({ headers: { Authorization: `Bearer ${accessToken}` } });
```

Comment:
공통 부분은 변수화 해주시는 것이 좋아보이네요!

```suggestion
    const headers = { Authorization: `Bearer ${accessToken}` }
    getFolders({ headers });
    getLinks({ headers });
```